packages/gestalt-design-tokens/tokens/vr-theme/sema/color/dark/default.json

```diff
@@ -117,15 +117,15 @@
         "wash": {
           "default": {
             "type": "color",
-            "value": "rgba({base.color.grayscale.500}, 0.8)"
```

Comment:
VR theme CANNOT BE touched

Reply:
chat with ravi if you have some blocker here

Reply:
Ok, reverted the changes, #3721 should fix the issue here